index.js

```diff
@@ -0,0 +1,84 @@
+const Base = require('bfx-facs-base')
+const { HyperCoreLogger } = require('hypercore-logs')
+const HyperCoreTransport = require('./lib/winston-transport-hypercore')
+const path = require('path')
+const winston = require('winston')
+
+class HyperCoreLoggerFacility extends Base {
+  constructor (caller, opts, ctx) {
+    super(caller, opts, ctx)
+
+    this.name = 'hypercore-logger'
+    this._hasConf = true
+
+    this.init()
+  }
+
+  _start0 (cb) {
```

Comment:
one thing I noticed, hypercore logger (writer) will be part of worker, if worker is down then if there's no other peer alive then you can't read logs, maybe it would be better to separate that part from worker, e.g. something like this:
- we could create a new logger for tcp similar to udp https://github.com/bitfinexcom/hypercore-logs/blob/master/src/hypercore-udp-logger.js
- and worker simply forwards all logs via tcp connection to the logger process
- we could do the same also via udp to avoid managing handshake and connection keep alive

Reply:
This is a great point!
In the current state we'll still get a big improvement of our debugging capabilities. But I think we can address this in the upcoming versions.